README.md

```diff
@@ -0,0 +1,99 @@
+# Seqera Labs Showcase Automation Scripts
+
+## Overview
+
+This repository contains automation scripts for launching and collecting information from pipelines in the Seqera Labs platform.
+
+### `launch_pipelines.py`
+
+This script performs the following steps:
+
+- Reads pipeline details from multiple YAML files in [`./pipelines/`](./pipelines/).
+- Reads compute environment details from multiple files in [`./compute-envs/`](./compute-envs/).
+- Creates combinations of pipelines and compute environments in an all-by-all manner.
+- Includes YAML files specified in [`include/`](./include/) as both pipelines and compute environments.
+- Excludes YAML files specified in [`exclude/`](./exclude/) based on pipeline and compute environment names.
+- Launches all pipelines generated from this data.
+- Logs any "failure to launch" without causing the program to fail.
+- Writes all pipelines to a YAML file for subsequent steps.
+
+### `collect_metadata.py`
```

Comment:
This is supposed to be `extract_metadata.py` or did you forget to rename the file to `collect_metadata.py`?